modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/DefinitionBuilder.js

```diff
@@ -63,7 +71,7 @@ export default function DefinitionBuilder(props) {
 		<DefinitionBuilderContextProvider {...contextProps}>
 			<div className="definition-builder-app">
 				<ReactFlowProvider>
-					<UpperToolbar {...props} />
+					<UpperToolbar {...props} blockingErros />
```

Comment:
@MariMalta there is a missing 'r' in the ending of the const name. Besides, its not necessary to be passed directly to UpperToolbar component because the const is being retrieved from DefinitionBuilder context inside UpperToolbar.